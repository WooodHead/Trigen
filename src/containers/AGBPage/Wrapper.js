import styled from 'styled-components';


export default styled.section`
  margin: 0 auto;
  width: 50%;
`;
