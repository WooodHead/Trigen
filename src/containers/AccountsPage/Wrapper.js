import styled from 'styled-components';


export default styled.section`
  min-width: 50%;
  margin: 0 auto;
`;
