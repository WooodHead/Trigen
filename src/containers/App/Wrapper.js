/**
  * Styled Wrapper for the whole application
  *
  * @namespace App.Wrapper
  * @memberOf App
  */

import styled from 'styled-components';


const Wrapper = styled.div`
  min-height: 100%;
  display: flex;
  flex-direction: column;
`;

export default Wrapper;
