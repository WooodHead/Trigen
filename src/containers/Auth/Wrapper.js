/**
  * Renders a styled div for the HomePage
  *
  * @namespace HomePage.Wrapper
  * @memberOf HomePage
  */

import styled from 'styled-components';


const Wrapper = styled.section`
  margin: 0 auto;
  width: 50%;
`;

export default Wrapper;
