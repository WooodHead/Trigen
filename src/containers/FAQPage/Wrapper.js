import styled from 'styled-components';


export default styled.section`
  width: 50%;
  margin: 0 auto;
`;

