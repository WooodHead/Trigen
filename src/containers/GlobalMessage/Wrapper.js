import styled from 'styled-components';


const Wrapper = styled.div`
  margin: 0 0 20px 0;
  padding: 10px 20px;

  border-radius: 4px;

  background: ${props => props.error ? props.theme.warningHover : props.theme.primaryHover};
  opacity: .6;
  color: #fff;
`;

export default Wrapper;
