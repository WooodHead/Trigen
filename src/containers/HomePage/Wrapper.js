/**
  * Renders a styled div for the HomePage
  *
  * @namespace HomePage.Wrapper
  * @memberOf HomePage
  */

import styled from 'styled-components';


const Wrapper = styled.section``;

export default Wrapper;
