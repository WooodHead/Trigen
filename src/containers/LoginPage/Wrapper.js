import styled from 'styled-components';


export default styled.div`
  height: 100%;

  .fullHeight {
    height: 100%;
  }

  .form {
    margin-top: 45px;
  }
`;
