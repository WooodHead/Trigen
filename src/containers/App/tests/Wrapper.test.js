import React from 'react';
import { shallow } from 'enzyme';

import Wrapper from '../Wrapper';


describe('<Wrapper />', () => {
  it('Should render section', () => {
    const wrapper = shallow(<Wrapper />);
    expect(wrapper.find('section').length).toMatchSnapshot();
  });
});
