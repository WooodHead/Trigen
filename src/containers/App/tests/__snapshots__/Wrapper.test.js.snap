exports[`<Wrapper /> Should render section 1`] = `1`;
